WordPressKit/WordPressComRestApi.swift

```diff
@@ -299,18 +299,54 @@ open class WordPressComRestApi: NSObject {
         return "\(String(describing: oAuthToken)),\(String(describing: userAgent))".hashValue
     }
 
-    fileprivate func buildRequestURLFor(path: String) -> String? {
-        let pathWithLocale = appendLocaleIfNeeded(path)
+    // AlamoFire our parameters via `SessionManager.request(_:method:parameters:encoding:headers:)`
```

Comment:
Is this comment missing a word somewhere near the beginning?

Reply:
Good catch - I have pushed an additional commit that removes the stray comment.